region/maridia/inner-pink/Botwoon Energy Tank Room.json

```diff
@@ -1998,6 +2111,90 @@
         {"shinespark": {"frames": 5, "excessFrames": 2}}
       ]
     },
+    {
+      "id": 53,
+      "link": [4, 7],
+      "name": "Suitless Water Shinecharge, Blue Speed Run",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 6,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "h_waterGetBlueSpeed"
+      ],
+      "clearsObstacles": ["A"],
+      "devNote": [
+        "This is just a rough estimate of the minimum number of tiles that this runway can represent.",
+        "It is designed for a skill level that won't be able to use canStutterWaterShineCharge."
+      ]
+    },
+    {
+      "id": 54,
+      "link": [4, 7],
+      "name": "Suitless Stutter Water Shinecharge",
+      "entranceCondition": {
+        "comeInStutterShinecharging": {
+          "minTiles": 2
+        }
+      },
+      "requires": [
+        "h_stutterWaterGetBlueSpeed"
+      ],
+      "clearsObstacles": ["A"]
+    },
+    {
+      "link": [4, 7],
+      "name": "Precise Suitless Stutter Water Shinecharge",
+      "entranceCondition": {
+        "comeInStutterShinecharging": {
+          "minTiles": 1
+        }
+      },
+      "requires": [
+        "h_stutterWaterGetBlueSpeed"
+      ],
+      "clearsObstacles": ["A"],
+      "note": [
+        "With only 1 tile of runway, this requires releasing forward for 1 or 2 frames",
+        "and repressing forward on the last possible frame."
+      ]
+    },
+    {
+      "id": 56,
+      "link": [4, 7],
+      "name": "Speedball",
+      "entranceCondition": {
+        "comeInSpeedballing": {
+          "runway": {
+            "length": 6,
+            "openEnd": 1
+          }
+        }
+      },
+      "requires": [
+        "canSpeedball"
+      ],
+      "note": [
+        "This is just a rough estimate of the minimum number of tiles that this runway can represent."
+      ]
+    },
+    {
+      "id": 58,
+      "link": [4, 7],
+      "name": "Blue Spring Ball Bounce",
+      "entranceCondition": {
+        "comeInWithBlueSpringBallBounce": {
+          "movementType": "controlled"
+        }
+      },
+      "requires": [],
+      "devNote": [
+        "With a mid-air spring ball jump it would be possible to go through the morph tunnel.",
```

Comment:
I don't get this note.
I think it still bounces through the speed blocks.

Reply:
Updated the note to try to make it more clear now. This strat does bounce through the speed blocks. The note is just saying that with these items you could already traverse the room in a simpler way by spring ball jumping up to the morph tunnel, rather than doing a blue spring ball bounce. The only logical advantage of the blue spring ball bounce is to circumvent the item in case you want to save it to be collected later.